app/src/main/res/drawable/ic_splash.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
+    <item
+        android:height="118dp"
+        android:width="118dp"
```

Comment:
It's better to set it to 100dp x 100dp. I found that on Pixel 3a API 33 the current sizes (118dp) cause a crop.
The guidelines suggests to use a vector icon, but a this time I don't have it.

Reply:
sorry for late response, 
I got them, I'm forgot to test the responsive for any screen, thanks  for suggestion